css/styles.css

```diff
@@ -139,30 +208,81 @@ td {
     transform: translate(-50%, -50%) scale(1);
     opacity: 1;
 }
+
 .card {
-    background:#c0c1c2;
+    background: #c0c1c2;
 }
+
 .card-body {
     text-align: left;
 }
 
+.card-subtitle {
+    color: #6c757d;
+    font-family: 'Open Sans', sans-serif;
+    font-size: 1rem;
+    font-style: normal;
+    font-weight: 700;
+}
+
+.card-text {
+    font-family: 'Open Sans', sans-serif;
+    font-size: 1rem;
+    font-style: normal;
+    font-weight: 400;
+}
+
+.card-link {
+    color: #5c6bcf;
+    font-family: 'Open Sans', sans-serif;
+    font-size: 1rem;
+    font-style: normal;
+    font-weight: 600;
+}
+
 
 /***************** EDUCATION ******************/
 
 .education {
```

Comment:
What is this for? education text, container, card? Add a more relevant class name